objects/src/accounts/code/mod.rs

```diff
@@ -50,13 +58,20 @@ impl AccountCode {
     /// Returns an error if:
     /// - Compilation of the provided module fails.
     /// - The number of procedures exported from the provided module is smaller than 1 or greater
-    ///   than 256.
+    ///   than 65535.
     pub fn new(module: ModuleAst, assembler: &Assembler) -> Result<Self, AccountError> {
         // compile the module and make sure the number of exported procedures is within the limit
         let procedures = assembler
             .compile_module(&module, None, &mut AssemblyContext::for_module(false))
             .map_err(AccountError::AccountCodeAssemblerError)?;
 
+        // TODO: Find way to input offset
+        let procedures: Vec<AccountProcedureInfo> = procedures
+            .into_iter()
+            .enumerate()
+            .map(|(i, proc)| AccountProcedureInfo::new(proc, i as u16))
+            .collect();
```

Comment:
We'll need to change this - but this can be done in the next PR.

Reply:
Yes will be changed.